### DISCUSSION TITLE
Importing save from another server

### DISCUSSION THREAD

**Discussion by Slothy2406** [factoriotools/factorio-docker#534] [Q&A]
HI,

I am just setting up a server using your docker image and I would like to import a save we have from another factorio server to test before we migrate the current server to a docker image. I have tried to overwrite the current autosave file in the /opt/factorio/saves folder but when we restart the server it doesnt pull in the current save. I have read the part about using the touch oldfile command but is this suppose to be executed within the docker container or just within the normal os?

thanks

Andy

**Comment by fank** [comment]
Can you please share the command of how you started the image?

**Comment by Slothy2406** [comment]
Initially ran the 
`sudo docker run -d -p 34197:34197/udp -p 27015:27015/tcp -v /opt/factorio:/factorio --name factorio --restart=unless-stopped factoriotools/factorio`

and then I run the docker stop/start image_id

As I am typing this, I am suddenly thinking that I should be re-running the first command rather than starting up the image.

I am a bit of a docker newb, and I may have jumped in right at the deep end with this!!

**Comment by Slothy2406** [comment]
Just tried to rebuild with the docker run, but that doesnt bring in the save file either

**Comment by Slothy2406** [comment]
ok, got it working, couple of things
1. make sure the permissions on the save file is correct, I had to chown the file to 845:845 once I had copied the file in.
2. I am not too sure if this makes much difference, but I made sure the folder in the zip file was the same as the name of the zip file (_autosave1.zip and _autosave1).

**Comment by JoeFri2k** [comment]
Hi, i am connected to the Server (docker) and try to move my saves to "/dockerdata/factorio/saves" but i do not have any permissions to copy saves in the Folder. I see that yo use chmod 845 for the rights. What i have to consider if i use sftp to transfer data (saves) to the server !?

**Comment by tpaktop77** [comment]
This is the workflow:

```
 unzip -X ./thisisgame.zip
 mv ./thisisgame ./_autosave1
 zip -r -X ./_autosave1.zip.2 ./_autosave1
 chmod --reference=./_autosave1.zip ./_autosave1.zip.2
 cp ./_autosave1.zip.2 ./_autosave1.zip
 docker-compose stop
 docker-compose up -d
```

**Comment by tpaktop77** [comment]
Well, replace `thisisthegame` with your name and also docker-compose commands with docker if you are using docker.
